rest/src/main/java/discord4j/rest/http/ReaderStrategy.java

```diff
@@ -42,9 +43,10 @@
      * Read from the input message and encode to a single object.
      *
      * @param response the response to read from
+     * @param content the response contents
      * @param responseType the type of object in the response which must have been previously checked via {@link
      * #canRead(Class, String)}
      * @return a Mono for the resolved response, according to the given response type
      */
-    Mono<Res> read(HttpClientResponse response, Class<Res> responseType);
+    Mono<Res> read(HttpClientResponse response, ByteBufFlux content, Class<Res> responseType);
```

Comment:
Is `response` still necessary? None of the implementations use it.

Reply:
Going to remove that argument. No need to keep it if we have everything under content.